libs/designsystem/tabs/src/tab-button/tab-button.component.spec.ts

```diff
@@ -1,24 +1,184 @@
-import { createComponentFactory, Spectator } from '@ngneat/spectator';
-import { MockComponent, MockModule } from 'ng-mocks';
-import { IonicModule } from '@ionic/angular';
+import { createHostFactory, Spectator } from '@ngneat/spectator';
+import { MockComponent } from 'ng-mocks';
 
+import { TestHelper } from '@kirbydesign/designsystem/testing';
+import { DesignTokenHelper } from '@kirbydesign/designsystem/helpers';
 import { IconComponent } from '@kirbydesign/designsystem/icon';
+
 import { TabButtonComponent } from './tab-button.component';
 
+const { getTextColor, fontSize, fontWeight, lineHeight, size } = DesignTokenHelper;
+
 describe('TabsComponent', () => {
   let spectator: Spectator<TabButtonComponent>;
+  let ionTabButton: HTMLIonTabButtonElement;
+  let innerButton: HTMLAnchorElement;
 
-  const createHost = createComponentFactory({
-    imports: [MockModule(IonicModule)],
+  const createHost = createHostFactory({
     component: TabButtonComponent,
+    imports: [TestHelper.ionicModuleForTest],
     declarations: [MockComponent(IconComponent)],
+    template: `<kirby-tab-button>Title</kirby-tab-button>`,
   });
 
-  beforeEach(() => {
-    spectator = createHost({});
+  describe('by default', () => {
+    beforeEach(async () => {
+      spectator = createHost();
+      ionTabButton = spectator.query('ion-tab-button');
+      await TestHelper.whenReady(ionTabButton);
+      ionTabButton.style.transition = 'none';
+      innerButton = ionTabButton.shadowRoot.querySelector('[part=native]');
+    });
+
+    it('should create', () => {
+      expect(spectator.component).toBeTruthy();
+    });
+
+    it('should set correct color on unselected tab', () => {
+      expect(ionTabButton).toHaveComputedStyle({
+        color: getTextColor('semi-dark'),
+      });
+    });
+
+    it('should set correct color on selected tab', () => {
+      ionTabButton.selected = true;
+
+      expect(ionTabButton).toHaveComputedStyle({
+        color: DesignTokenHelper.getTextColor('semi-dark'),
```

Comment:
```suggestion
        color: getTextColor('semi-dark'),
```

Reply:
Just double checking - should `color` be `semi-dark` both when the tab is unselected and selected?

Reply:
> Just double checking - should `color` be `semi-dark` both when the tab is unselected and selected?

Nope, that's a copy-paste mistake 🙈  - good catch! Fixed 👍🏻